data/scripts/creaturescripts/player/login.lua

```diff
@@ -0,0 +1,162 @@
+local function sendBoostMessage(player, category, isIncreased)
+	return player:sendTextMessage(MESSAGE_BOOSTED_CREATURE, string.format(isIncreased and "Event! %s is increased. Happy Hunting!" or "Event! %s is decreased. Happy Hunting!", category))
```

Comment:
essa forma do string.format está redundante. o ideal é usar o texto completo, e só incluir as diferenças.
```suggestion
	return player:sendTextMessage(MESSAGE_BOOSTED_CREATURE, string.format("Event! %s is %screased. Happy Hunting!", category, isIncreased and "in" or "de"))
```